dotnet/src/Connectors/Connectors.Memory.Chroma/ChromaClientException.cs

```diff
@@ -1,45 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-
-using System;
-using System.Globalization;
-
-namespace Microsoft.SemanticKernel.Connectors.Memory.Chroma;
-
-/// <summary>
-/// Exception to identify issues in <see cref="ChromaClient"/> class.
-/// </summary>
-public class ChromaClientException : Exception
-{
-    private const string CollectionDoesNotExistErrorFormat = "Collection {0} does not exist";
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="ChromaClientException"/> class.
-    /// </summary>
-    public ChromaClientException() : base()
-    {
-    }
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="ChromaClientException"/> class.
-    /// </summary>
-    /// <param name="message">The message that describes the error.</param>
-    public ChromaClientException(string message) : base(message)
-    {
-    }
-
-    /// <summary>
-    /// Initializes a new instance of the <see cref="ChromaClientException"/> class.
-    /// </summary>
-    /// <param name="message">The message that describes the error.</param>
-    /// <param name="innerException">Instance of inner exception.</param>
-    public ChromaClientException(string message, Exception innerException) : base(message, innerException)
-    {
-    }
-
-    /// <summary>
-    /// Checks if Chroma API error means that collection does not exist.
-    /// </summary>
-    /// <param name="collectionName">Collection name.</param>
-    public bool CollectionDoesNotExistException(string collectionName) =>
```

Comment:
Not sure if this method should be the responsability of this exception, maybe a property?